lib/trie/trie.go

```diff
@@ -892,7 +893,7 @@ func (t *Trie) clearPrefix(parent *Node, prefix []byte) (
 	noPrefixForNode := len(prefix) <= len(branch.Key) ||
 		lenCommonPrefix(branch.Key, prefix) < len(branch.Key)
 	if noPrefixForNode {
-		const nodesRemoved = 0
+		const nodesRemoved = 0 // XXX
```

Comment:
```suggestion
		const nodesRemoved = 0 // XXX
```
`XXX`?

Reply:
Oopsi

For context, I used that to see test coverage and see where that bug was coming from, since some of the tests are fuzzing-like and generate random values.